iotqatools/mongo_utils.py

```diff
@@ -209,6 +209,79 @@ def drop_database(self):
         except Exception, e:
             assert False, " ERROR - Deleting a database %s in MongoDB...\n %s" % (self.current_collection, str(e))
 
+    def get_all_databases(self):
+        """
+        Get all databases in mongo
+        :return: Database array found
+        """
+        arr_dbs = self.client.database_names()
+        return arr_dbs
```

Comment:
I recommend to apply `get_cursor_value` method and return a list instead of a cursor